package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "csv-stringify-as-promised",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "csv-stringify wrapped in a promise",
   "main": "lib/index.js",
   "scripts": {
-    "lint": "standard | snazzy",
+    "lint": "./node_modules/.bin/eslint lib test",
+    "test-mocha": "./node_modules/.bin/mocha",
```

Comment:
You don't need to specify `./node_modules/.bin` in npm scripts. npm automatically looks for those binaries when doing `npm run ...`

Reply:
but it can use the global dependencies which may be in other versions